src/gui.cc

```diff
@@ -494,6 +494,21 @@ void GUI::DisplayTimeline(
 
     TogglTimelineChunkView *first_chunk = nullptr;
 
+    // Get all entires in this day (no chunk, no overlap)
+    TogglTimeEntryView *first_entry = nullptr;
+    time_t start_day = datetime.timestamp().epochTime();
+    time_t end_day = start_day + 86400; // one day
+    for (unsigned int i = 0; i < entries_list.size(); i++) {
```

Comment:
Get all TEs in this selected day from library, then passing to the app. The app will take it over, and determine if the TE is overlapping with other or not.